src/loader.js

```diff
@@ -271,8 +271,10 @@ function pitch(request) {
           const exportsString = `export { ${identifiers
             .map(([id, key]) => `${id} as ${JSON.stringify(key)}`)
             .join(", ")} }`;
-
-          return `${localsString}\n${exportsString}\n`;
+          const exportDefaultString = `export default ${JSON.stringify(
+            locals
+          )}`;
```

Comment:
```suggestion
          const exportDefaultString = `export default { ${identifiers
            .map(([id, key]) => `${JSON.stringify(key)}: ${id}`)
            .join(", ")} }`;
```

That would be a bit more efficient

Reply:
done:

```diff
--- a/test/cases/es-named-export-as-is/expected/main.js
+++ b/test/cases/es-named-export-as-is/expected/main.js
@@ -17,7 +17,7 @@ var _1 = "Xh041yLR4iCP4RGjge50";
 var _2 = "NMuRsxoDwvW8BhSXhFAY";
 var _3 = "ayWIv09rPsAqE2JznIsI";
 
-/* harmony default export */ const __WEBPACK_DEFAULT_EXPORT__ = ({"a-class":"Xh041yLR4iCP4RGjge50","b__class":"NMuRsxoDwvW8BhSXhFAY","cClass":"ayWIv09rPsAqE2JznIsI"});
+/* harmony default export */ const __WEBPACK_DEFAULT_EXPORT__ = ({ "a-class": _1, "b__class": _2, "cClass": _3 });
```

could this somehow prevent the minifier from optimizing?